### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by aleksanderchan**
Setting things up, planning, getting started on the array of objects (songs) that will be used for testing.

### CODE REVIEW

File: js/einSearch.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// 	}
+
+//IN THE FUTURE... I want artists and genres to be referenced in these songs. They should be their own entities. Shit should look like this:
+// {
+//		"title": "Some Title",
+//		"artist": "someurl.com/artists/someId"
+//		"genre": "someurl.com/genres/someId"
+// }
+
+musicLib = [
```
Comment (t4ks) [inline js/einSearch.js]: musicLib is a variable that is assigned an array so ```var musicLib``` please, or else it'll be defined as a global, which is generally bad practice.

File: js/einSearch.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		"artist": "Daft Punk",
+		"genre": "Electronic"
+	},
+
+	{
+		"title": "Man in the Mirror",
+		"artist": "Michael Jackson",
+		"genre": "Pop"
+	}
+]
```
Comment (t4ks) [inline js/einSearch.js]: missing semicolon ``` ]; ```

File: js/einSearch.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+user will search for shit
+whatever is returned in the search will be created as DOM elements.
```
Comment (t4ks) [inline js/einSearch.js]: comments must have ``` // ``` otherwise error.

File: js/einSearch.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+//  	"artist":
+//		"genre":
+// 	}
+
+//IN THE FUTURE... I want artists and genres to be referenced in these songs. They should be their own entities. Shit should look like this:
+// {
+//		"title": "Some Title",
+//		"artist": "someurl.com/artists/someId"
+//		"genre": "someurl.com/genres/someId"
+// }
```
Comment (t4ks) [inline js/einSearch.js]: I like this plan.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+
```
Comment (t4ks) [inline index.html]: why you no ```<html>```

**Comment by aleksanderchan** [comment]
Fixed syntax errors

**Comment by taylor-kisor-smith** [comment]
ship it

**Pull request merged**